data/base-spells.yaml

```diff
@@ -1596,6 +1601,16 @@ spell_data:
     abbrev: PD
     mana: 2
     mana_type: lunar
+  Pattern Hues:
+    abbrev: p h
+    prep_type: prep cantrip
+    prep_time: 0
+    cast: gesture phosphorescent emerald green
+    mana:
+    harmless: true
+    expire: The faint .* aura fades from around you.
+    recast: 1
```

Comment:
This should be a negative number or 0, as this is only used by the system when a spell from the Perc window is about to expire. It doesn't technically matter as it's not going to be used really, but it may confuse people.

Reply:
Testing -1 for recast.